File: perl/euler1.pl
#!/usr/bin/perl
# Euler1 in Perl

sub euler1 {
	my @array = (0..shift);
	my $sum = 0;
	foreach (@array) {
		if ($_ % 3 == 0 or $_ % 5 == 0) {
			$sum += $_;
		}
	}
	return $sum;
}
print euler1(999);

File: php/euler1.php
#!/usr/bin/php -q
<?php
// Euler1 in PHP

function euler1($n) {
    $sum = 0;

    for ($i=0; $i<$n; $i++) {
        if ($i%3 == 0 || $i%5==0) {
            $sum += $i;
        }
    }
    return $sum;
}

$result = euler1(1000);
?>
Euler1 = <?php echo $result . "\n" ?>
